autoload/echodoc/util.vim

```diff
@@ -0,0 +1,148 @@
+" Returns a parsed stack of functions found in the text.  Each item in the
```

Comment:
Hmm, I was going to ask if you wanted to change the header style to be like deoeplete.nvim and move the license to a separate LICENSE file.